host-spec/c06-consensus.tm

```diff
@@ -420,7 +525,7 @@
 
     \;
 
-    <math|H<rsub|BABE><around*|(|B|)>> must be included as a diegst item of
+    <math|H<rsub|BABE><around*|(|B|)>> must be included as a digest item of
     Pre-Runtime type in the header digest <math|H<rsub|d><around*|(|B|)>> as
     defined in Definition <reference|defn-digest>.\ 
   </definition>
```

Comment:
what is H_h(B)?

Reply:
In Algorithm 6.3., SLOT_TIME(s) is not defined and threshold \tau should be written as defined in Definition 6.8

Reply:
> what is H_h(B)?

It is the block header hash, but I also added a note to the definition of the block signature to  make this clearer.

Reply:
> In Algorithm 6.3., SLOT_TIME(s) is not defined

Indeed I seem overlooked that when changing the name of the median algorithm definition in 6.2. I will rework the definition to have that fixed.

> threshold \tau should be written as defined in Definition 6.8

Do you mean adding the subscript or replacing \tau with its definition?

Reply:
Added back Slot-Time and updated threshold.